vtex-apps/tiktok-tbp
# BRPA-713 pixel structure

#### What is the purpose of this pull request?
- Add the pixel builder and create the basic structure to load the TikTok pixel.

#### How to test it?
In [this workspace](https://tiktok--sandboxbrdevbrl.myvtex.com/). You can change the pixel id in the app settings.
When there is a valid pixel ID, the app will initialize the TikTok pixel (you can check that with the TikTok Pixel Helper extension for chrome). Otherwise, a warning message will be shown on the browser console.

#### Screenshots or example usage
<img width="1440" alt="Screen Shot 2022-01-04 at 14 47 23" src="https://user-images.githubusercontent.com/1629129/148101705-a406f080-24da-4e90-adf9-1d11a914314e.png">

#### How does this PR make you feel? [:link:](http://giphy.com/)

![](https://media.giphy.com/media/xULW8vRQrlIPRfiEog/giphy.gif)

#### Types of changes
